__mocks__/src/statsig/index.ts

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Looks like this may have broken getExperimentParams mocks in various test files. Any reason to have these generic mocks?